### PR TITLE
Close RCgNMI instance in smoke tests

### PR DESCRIPTION
**Pull request by PeterSuna**
Close RcGNMI instance if initialization fails.
Close RcGNMI instance in negative test.
Fix typo

ID: 1847

### CODE REVIEW

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java [hunk 1]
```diff
@@ -8,32 +8,34 @@
...
 
     @Test
     public void gnmiModuleSmokeTest() throws ConfigurationException {
-        final RcGnmiAppModule module = new RcGnmiAppModule(RcGnmiAppModuleConfigUtils.loadDefaultConfig(),
+        final var module = new RcGnmiAppModule(RcGnmiAppModuleConfigUtils.loadDefaultConfig(),
                 Executors.newCachedThreadPool(), MODULE_TIMEOUT, null);
-        Assertions.assertTrue(module.initModules());
-        Assertions.assertTrue(module.close());
+        final var isModuleInitSuccessfully = module.initModules();
+        assertTrue(module.close());
```
Comment (ihrasko) [inline lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java:28]:
```
assertTrue(module.initModules());
assertTrue(module.close());

```

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java [hunk 2]
```diff
@@ -8,32 +8,37 @@
...
 
 import io.lighty.core.controller.impl.config.ConfigurationException;
 import java.util.concurrent.Executors;
-import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
 
 public class RcGnmiAppModuleTest {
     private static final int MODULE_TIMEOUT = 60;
+    private static RcGnmiAppModule rcgnmiModule;
```
Comment (ihrasko) [inline lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java:23]: No need to be static.

**Review by ihrasko — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java [hunk 3]
```diff
@@ -8,32 +8,34 @@
...
     public void gnmiModuleStartFailedTest() throws ConfigurationException {
-        final RcGnmiAppConfiguration config = spy(RcGnmiAppModuleConfigUtils.loadDefaultConfig());
+        final var config = spy(RcGnmiAppModuleConfigUtils.loadDefaultConfig());
         when(config.getControllerConfig()).thenReturn(null);
-        final RcGnmiAppModule rgnmiAppModule = new RcGnmiAppModule(config, Executors.newCachedThreadPool(),
-                MODULE_TIMEOUT, null);
-        Assertions.assertFalse(rgnmiAppModule.initModules());
+        final var rcgnmiAppModule = new RcGnmiAppModule(config, Executors.newCachedThreadPool(), MODULE_TIMEOUT, null);
+        final var isStarted = rcgnmiAppModule.initModules();
+        rcgnmiAppModule.close();
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java]: Close, moved to AfterEach method

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java (hunk 1 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java]: Close, moved to AfterEach method

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java (hunk 2 above)
Comment (PeterSuna) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/test/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModuleTest.java]: Removed

**Review by ihrasko — APPROVED** [review]

**Pull request merged**